sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py

```diff
@@ -216,7 +216,7 @@ def __init__(
         super(Input, self).__init__(type=type)
         # As an annotation, it is not allowed to initialize the name.
         # The name will be updated by the annotated variable name.
-        self.name = None
+        self.arg_name = None
```

Comment:
Why changed this? Where will `self.arg_name` be used?

Reply:
I find that some tests will assign name for input in previous code. This may be the reason why define self.name here. 
![image](https://user-images.githubusercontent.com/49266998/215384706-25eb7fa7-3f77-48e6-a89d-a542774db454.png)
But currently the name and version would take other role in Classs Output and in Class Output the previous name is renamed to `arg_name`. I think the previous name in Class Input and Class Input should keep the same, thus I modify here

Reply:
replaced with `_arg_name`